source/fundamentals/connection.txt

```diff
@@ -478,13 +466,6 @@ parameters of the connection URI to specify the behavior of the client.
      - Specifies that the driver must retry supported read operations
        if they fail due to a network error. Defaults to ``true``.
 
-   * - **uuidRepresentation**
```

Comment:
In the [Server Docs](https://docs.mongodb.com/manual/reference/connection-string/#mongodb-urioption-urioption.uuidRepresentation) it says that Not all Drivers support this option, so that could be an explanation for why it is not included in the Driver Spec (not all drivers need to support it, so it is not a criteria of a valid driver). I think this should stay as an option[ as this feature is in the driver here](http://mongodb.github.io/mongo-java-driver/4.3/apidocs/bson/org/bson/UuidRepresentation.html?is-external=true), however whether you go keep this or delete it I think the external reviewer should be asked to specifically check if this option should be discussed in the Java Docs (this is a big PR and this is a subtle change so wouldn't want it to slip through without a Driver Engineer considering this option thoroughly).

Reply:
Agreed that this needs to stay.